packages/core/test/integration/pool.test.ts

```diff
@@ -41,6 +41,10 @@ function assertSameConnection(
       expect(newConnection.dummyId).to.equal(oldConnection.dummyId).and.to.be.ok;
       break;
 
+    case 'oracle':
+      expect(oldConnection).to.be.equal(newConnection);
```

Comment:
Is there no id we can use here? Also; it would be good to include the `.and.to.be.ok` here

Reply:
If we were using Oracle's connection pool, then we may have some id. Added `.and.to.be.ok`